resources/lang/id/home.php

```diff
@@ -0,0 +1,152 @@
+            'title' => 'Beatmaps',
+        ],
+
+        'forum_post' => [
+            'all' => 'Semua forum',
+            'link' => 'Cari di forum',
+            'more_simple' => 'Lihat hasil pencarian forum lainnya',
+            'title' => 'Forum',
+
+            'label' => [
+                'forum' => 'cari di forum',
+                'forum_children' => 'termasuk subforum',
+                'topic_id' => 'topik #',
+                'username' => 'pemilik',
+            ],
+        ],
+
+        'mode' => [
+            'all' => 'semua',
+            'beatmapset' => 'beatmap',
+            'forum_post' => 'forum',
+            'user' => 'pemain',
+            'wiki_page' => 'wiki',
+        ],
+
+        'user' => [
+            'more' => ':count hasil pencarian pengguna lainnya',
+            'more_simple' => 'Lihat hasil pencarian pengguna lainnya',
+            'more_hidden' => 'Pencarian pengguna terbatas untuk :max pengguna saja. Perbaiki kata kunci pencarian Anda',
+            'title' => 'Pengguna',
+        ],
+
+        'wiki_page' => [
+            'link' => 'Cari di wiki',
+            'more_simple' => 'Lihat hasil pencarian wiki lainnya',
+            'title' => 'Wiki',
+        ],
+    ],
+
+    'download' => [
+        'tagline' => 'mari persiapkan<br>diri Anda!',
+        'action' => 'Unduh osu!',
+        'os' => [
+            'windows' => 'untuk Windows',
+            'macos' => 'untuk macOS',
+            'linux' => 'untuk Linux',
+        ],
+        'mirror' => 'alternatif',
+        'macos-fallback' => 'pengguna macOS',
+        'steps' => [
+            'register' => [
+                'title' => 'buat akun',
+                'description' => 'ikuti petunjuk saat memulai permainan untuk masuk atau membuat akun baru',
+            ],
+            'download' => [
+                'title' => 'unduh permainannya',
+                'description' => 'klik tombol di atas untuk mengunduh penginstal, lalu jalankan!',
+            ],
+            'beatmaps' => [
+                'title' => 'dapatkan beatmaps',
+                'description' => [
+                    '_' => ':browse koleksi beatmap yang luas buatan pengguna dan mulai bermain!',
+                    'browse' => 'telusuri',
+                ],
+            ],
+        ],
+        'video-guide' => 'panduan video',
+    ],
+
+    'user' => [
+        'title' => 'dashbor',
+        'news' => [
+            'title' => 'Berita',
+            'error' => 'Kesalahan memuat berita, coba muat ulang halaman?...',
+        ],
+        'header' => [
+            'welcome' => 'Halo, <strong>:username</strong>!',
+            'messages' => 'Anda memiliki :count pesan baru|Anda memiliki :count pesan baru',
+            'stats' => [
+                'friends' => 'Teman yang Daring',
+                'games' => 'Permainan',
+                'online' => 'Pemain yang Daring',
+            ],
+        ],
+        'beatmaps' => [
+            'new' => 'Beatmap yang Baru Diakui',
```

Comment:
while I very much understand what you're trying to translate into here I don't think "Diakui" is a good equivalent of "Approved" here - since basically that panel in question only shows newly ranked maps perhaps try something like this : 

+            'new' => 'Beatmap Berstatus Ranked Terbaru',

Reply:
The original line for this was:
>New Approved Beatmaps

Should i do "Beatmap Berstatus Approved Terbaru" or follow yours?

Reply:
"Approved Beatmap" is actually a term that in the past was used to refer to "marathon beatmaps" that wasn't rankable according to the old Ranking Criteria back then (back then when osu! still uses Ranked Score as its main metric the RC prohibits any marathon map or any map that is having >20mils in Ranked Score to be Ranked, hence the "Approved" category was created in order to facilitate maps whose scores won't count towards the user's total Ranked Score but still deemed good enough by the entire nominating entities - see http://osu.ppy.sh/beatmapsets/27148#osu/92087 for example)

Nowadays this rule does not apply anymore though, basically any map that has been through from the entire qualification process will be listed as "Ranked" regardless of their maximum score or their length (see http://osu.ppy.sh/beatmapsets/753955#osu/1587272 for example :P)

Reply:
Aha roger that.
Will make the changes.

Reply:
Roger that, will apply the change.